pkg/util/httputil/client.go

```diff
@@ -63,8 +63,8 @@ type Client struct {
 
 // Get does like http.Get but uses the provided context and obeys its cancellation.
 // It also uses the default client with a default 3 second timeout.
-func Get(ctx context.Context, url string) (resp *http.Response, err error) {
-	return DefaultClient.Get(ctx, url)
+func Get(ctx context.Context, url string, h *http.Header) (resp *http.Response, err error) {
```

Comment:
Assuming that you're adding `GetWithHeaders` to avoid breaking old code that uses `Get`, I don't think we should change the function signature here. It should just call `DefaultClient.Get` and you can add a `GetWithHeaders` function separately if needed. That should get rid of the diffs in non roachprod/roachtest stuff.

Reply:
I initially thought about this, but given that the usage was less, I modified the method signature rather than maintaining 2 methods. Also, others like Put does not have 2 methods.